### ISSUE TITLE
Title Quantum Technology 2023 proposal

### ISSUE THREAD

**Issue by alaarman** [alaarman/quantumtech2023#1]
Hi @timcp

I created this empty repo because we need a title. I am thinking to write the proposal around our KR paper.
Do you think this is viable? Protocol verification could be an alternative idea.
Do you have ideas for a title? Unfortunately, we cannot change it. So maybe rather something vague and catchy (that starts with an A ;))

This is the call:
https://www.nwo.nl/en/calls/quantum-technology-2023
We would fit into CAT-1 Quantum Algorithms or Quantum Internet.

**Comment by timcp** [comment]
The projects assigned last year were all quite close to implementation (i.e. "real" quantum hardware in the lab). That could be a coincidence, but maybe not. Something to keep in mind when writing the proposal.

> I am thinking to write the proposal around our KR paper.

Did you have anything concrete in mind on these topics? I put all DD ideas I had in the proposal from last year.

> Do you think this is viable?

Yes, provided the idea is more concrete. (and of course we can spend enough time on this)

> Do you have ideas for a title?

Is the deadline sooner than 13 april? At first skim I only saw that deadline as final deadline at the nwo web page

**Comment by alaarman** [comment]
Yes because we need an embedding declaration from ULeiden. So I need a title this week.

So your proposal was applied because it studied noise in practice? So maybe topology optimization using the methods we study in KR?

**Comment by alaarman** [comment]
De call stelt ook expliciet: het vinden van algoritmen waar QC wint. Wat als we dit practisch aanpakken: in het heuristische domain.
- We kunnen bijv SAT solver heuristieken implementeren met quantum tree search
- We kunnen kijken of er argumemnten zijn waarom het Google paper over quartic speedups (ze stellen dat Grover's kwadratische speedup niet genoeg is) het mis heeft, bijv omdat ze aannemen dat problemen te parallelliseren zijn en dat is niet altijd het geval (P-complete problemen)
- We kunnen de KR paper technieken inzetten om te kijken waar klassieke simulatie de mist in gaat.

**Comment by timcp** [comment]
> Yes because we need an embedding declaration from ULeiden. So I need a title this week.

Ah, I see. (Deze week al? Is twee weken van tevoren niet voldoende? Dat geeft ons ruimte om te overleggen.)

Dank voor je voorstellen. Als ik in de commissie zou zitten, dan zou ik de 'positieve' voorstellen (bij gebrek aan een beter woord) het beste vinden (topology optimization to make a QC work, combining classical heuristics with the power of quantum), liever dan de rest (als ik even alle nuances vergeet en sterk chargeer, dan leest het Google project bijvoorbeeld als: we laten zien dat alle slimme mensen bij Google het mis hebben. Die zouden we dan goed moeten fraseren)


>> Do you think this is viable?
> Yes, provided the idea is more concrete. (and of course we can spend enough time on this)

Ik kom hier op terug: wat precies verwacht je van mij? Ik heb vrijwel alle andere projecten en afspraken uitgesteld voor het KR paper (en in mindere mate voor het LIMDD paper), inclusief een andere mogelijke deadline eind maart voor een extern project. Dus ik heb een hoop in te halen na afloop van de KR deadline.


En is je voorstel om er een PhD student mee aan te nemen? Of een postdoc?
